src/config/sequelize.ts

```diff
@@ -6,9 +6,9 @@ const DEFAULT = {
     let dialect = "postgres";
     let host = "127.0.0.1";
     let port = "5432";
-    let database = "actionhero";
-    let username = undefined;
-    let password = undefined;
+    let database = "ahtodotutorial";
+    let username = "ymmy";
+    let password = "";
```

Comment:
I think hard-coding a database name is OK if you tell people to create a database of the same name in the README.  I'd remove the name here.  Postgres, by default, will use the currently logged in user, which is normally a safe default.

Reply:
Removed this for now. I've added in the description about replacing it with their own configurations.